skrub/tests/test_table_vectorizer.py

```diff
@@ -684,6 +684,8 @@ def test_accept_pipeline():
 
 def test_clean_null_downcast_warning():
     # non-regression test for https://github.com/skrub-data/skrub/issues/894
+    if parse_version(sklearn.__version__) < parse_version("1.4"):
```

Comment:
It would be best to decorate the test:

```python
pytest.mark.skipif(
    parse_version(sklearn.__version__) < parse_version("1.4"),
    reason="xxx"
)
def test_...
```

Reply:
thanks. I'll keep that in mind for next time. here as we've just decided to remove the warning we don't need to skip the test anymore